index.js

```diff
@@ -1,10 +1,7 @@
 var isNpmNotYarn = require('is-npm-not-yarn')
-var fs = require('fs')
-var path = require('path')
 
 function useYarn(message) {
   if (isNpmNotYarn && process.env.DISABLE_USE_YARN !== 'true') {
-    message = message || fs.readFileSync(path.resolve(__dirname, 'message.txt'), 'utf8')
```

Comment:
I don't think we should remove support for the default message with the programmatic API, that's a breaking change.

Reply:
Will restore it!